usr.bin/gh-bc/Makefile

```diff
@@ -55,7 +55,7 @@ MAN_SRC_DC=	dc/A.1
 
 # prevent floating point incompatibilities caused by -flto on some architectures
 .if !${MACHINE_ARCH:Mmips*} && ${MACHINE_ARCH:Mpowerpc64*} && \
-    !${MACHINE_ARCH:Mriscv64*}
+    !${MACHINE_ARCH:Mriscv64*} && !${MACHINE_ARCH:Maarch64*}
```

Comment:
Sort

Reply:
Done.  The powerpc64 case is missing the ! in our local diff.